src/Microsoft.IdentityModel.JsonWebTokens/JwtTokenUtilities.cs

```diff
@@ -319,35 +319,42 @@ private static long ParseTimeValue(JToken jToken, string claimName)
         /// <summary>
         /// Validates the 'typ' claim of the JWT token header.
         /// </summary>
-        /// <param name="type">The value of the 'typ' header claim."/></param>
+        /// <param name="type">The value of the 'typ' header claim.</param>
+        /// <param name="token">The security token.</param>
         /// <param name="validationParameters"><see cref="TokenValidationParameters"/> required for validation.</param>
         /// <exception cref="ArgumentNullException">If <paramref name="validationParameters"/> is null or whitespace.</exception>
         /// <exception cref="SecurityTokenInvalidTypeException">If <paramref name="type"/> is null or whitespace and <see cref="TokenValidationParameters.ValidTypes"/> is not null.</exception>
         /// <exception cref="SecurityTokenInvalidTypeException">If <paramref name="type"/> failed to match <see cref="TokenValidationParameters.ValidTypes"/>.</exception>
         /// <remarks>An EXACT match is required. <see cref="StringComparison.Ordinal"/> (case sensitive) is used for comparing <paramref name="type"/> against <see cref="TokenValidationParameters.ValidTypes"/>.</remarks>
-        internal static void ValidateTokenType(string type, TokenValidationParameters validationParameters)
+        internal static string ValidateTokenType(string type, SecurityToken token, TokenValidationParameters validationParameters)
```

Comment:
FWIW, this method doesn't have any JWT-specific logic, so it could be moved to `Validators.cs`.

Reply:
I agree it should be in Validators.cs